2-mandatory/4-stocks.js

```diff
@@ -48,7 +60,21 @@ function getAveragePrices(closingPricesForAllStocks) {
     The price change value should be rounded to 2 decimal places, and should be a number (not a string)
 */
 function getPriceChanges(closingPricesForAllStocks) {
-    // TODO
+  // TODO
+  let weekOpeniningPrice;
+  let weekClosingPrice;
+  let changeInPrice;
+  let stockPricesChange = [];
+  for (let i = 0; i < STOCKS.length; i++) {
+    weekOpeniningPrice = CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS[i][0];
+    weekClosingPrice = CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS[i][4];
+
+    changeInPrice = weekClosingPrice - weekOpeniningPrice;
+    changeInPrice = Number(changeInPrice.toFixed(2));
+    stockPricesChange[i] = changeInPrice;
+  }
+  console.log(stockPricesChange, "----------------------> stockPricesChange");
```

Comment:
Don't forget to remove console.log statements when done if they're just for debugging :ok_hand:

Reply:
Oh, yeah, I forgot to clean it out; thanks for your thorough review. :-)